geo/src/algorithm/bounding_rect.rs

```diff
@@ -141,6 +142,61 @@ where
     }
 }
 
+impl<T> BoundingRect<T> for Geometry<T>
+where
+    T: CoordinateType,
+{
+    type Output = Option<Rect<T>>;
+
+    fn bounding_rect(&self) -> Self::Output {
+        match self {
+            Geometry::Point(_) => None,
```

Comment:
Won't this give the wrong answer for some collection that include points?

e.g.
```
Collection = [ 
  Point (0, 0), 
  Rect { (1, 1), (2, 2) }
  Rect { (2, 2), (3, 3) }
] 
```

I would expect the bounding_rect to be `Rect { (0, 0), (3, 3) }`

But the current impl seems like it would return `Rect { (1, 1), (3, 3) }`

It seems like we could solve this by having point.bounding_rect() return a zero area rect (`Rect { point, point }` )

Reply:
addressed in https://github.com/georust/geo/pull/443/commits/83c90abfcef1a072181d10ce731fe0a48891c800